rllib/policy/torch_policy.py

```diff
@@ -315,7 +315,15 @@ def compute_actions_from_input_dict(
                 input_dict[k] for k in input_dict.keys() if "state_in" in k[:8]
             ]
             # Calculate RNN sequence lengths.
-            seq_lens = np.array([1] * len(input_dict["obs"])) if state_batches else None
+            seq_lens = (
+                torch.tensor(
+                    [1] * len(input_dict["obs"]),
+                    dtype=torch.long,
+                    device=input_dict["obs"].device,
```

Comment:
this looks fine to me. Is there a reason why before it was a numpy array? was it eventually getting converted to a torch tensor?

Reply:
No it wasn't, sometimes the model gets a numpy array instead. `np.ndarrray.max` and `torch.Tensor.max` work identically, so this is not an issue in `rnn_sequencing.py`, for example. But in my custom models, I need to explicitly check and cast `seq_lens` to `torch.Tensor` if it is `np.ndarray`.